src_py/sprite.py

```diff
@@ -416,7 +450,7 @@ def copy(self):
 
         """
         return self.__class__(  # noqa pylint: disable=too-many-function-args
-            self.sprites()  # Needed because copy() won't work on AbstractGroup
+            self._spritelist  # Needed because copy() won't work on AbstractGroup
```

Comment:
Copy says we need a copy, but the change seems to contradict that (`self.spites()` would trigger a copy AFAICT)

Reply:
... actually, I am not sure what the comment is trying to say, but it does say `copy()` which smells like it wanted a copy.  If the comment is outdated (or could be worded better and you have an idea what it means - I do not), then I think it should be updated as well.

Reply:
Yeah I'm not sure what that comment it is about. 

AFAICT things seem to work fine passing the raw list in here, you get a new Group with a new list containing the same sprites. 

I'll tinker around with copy a bit more I think, but either way it isn't important to change this anyway so I can revert it to calling `sprites()`.